<ui version="4.0" >
 <class>SMCStyleWidget</class>
 <widget class="QWidget" name="SMCStyleWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>366</width>
    <height>478</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <layout class="QHBoxLayout" >
     <property name="spacing" >
      <number>6</number>
     </property>
     <property name="margin" >
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="parentLabel" >
       <property name="text" >
        <string>Based On:</string>
       </property>
       <property name="wordWrap" >
        <bool>false</bool>
       </property>
       <property name="buddy" >
        <cstring>parentCombo</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="parentCombo" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Minimum" >
         <horstretch>5</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="basicGroup" >
     <property name="title" >
      <string>Basic Formatting</string>
     </property>
     <layout class="QVBoxLayout" >
      <item>
       <widget class="SMFontComboH" native="1" name="fontFace_" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize" >
         <size>
          <width>0</width>
          <height>62</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" >
        <item>
         <widget class="QLabel" name="fontSizeLabel_" >
          <property name="minimumSize" >
           <size>
            <width>22</width>
            <height>22</height>
           </size>
          </property>
          <property name="text" >
           <string/>
          </property>
          <property name="buddy" >
           <cstring>fontSize_</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="SMScrSpinBox" name="fontSize_" >
          <property name="minimum" >
           <number>1</number>
          </property>
          <property name="maximum" >
           <number>2048</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="trackingLabel_" >
          <property name="minimumSize" >
           <size>
            <width>22</width>
            <height>22</height>
           </size>
          </property>
          <property name="text" >
           <string/>
          </property>
          <property name="buddy" >
           <cstring>tracking_</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="SMScrSpinBox" name="tracking_" >
          <property name="suffix" >
           <string> %</string>
          </property>
          <property name="minimum" >
           <number>-300</number>
          </property>
          <property name="maximum" >
           <number>300</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="widthSpaceLabel" >
          <property name="text" >
           <string>TextLabel</string>
          </property>
          <property name="buddy" >
           <cstring>widthSpaceSpin</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="SMScrSpinBox" name="widthSpaceSpin" >
          <property name="suffix" >
           <string> %</string>
          </property>
          <property name="minimum" >
           <number>1</number>
          </property>
          <property name="maximum" >
           <number>199</number>
          </property>
          <property name="value" >
           <number>100</number>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" >
        <item>
         <widget class="SMStyleSelect" native="1" name="effects_" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Fixed" hsizetype="Minimum" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize" >
           <size>
            <width>30</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType" >
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <spacer>
        <property name="orientation" >
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType" >
         <enum>QSizePolicy::MinimumExpanding</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>300</width>
          <height>0</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="advGroup" >
     <property name="title" >
      <string>Advanced Formatting</string>
     </property>
     <layout class="QVBoxLayout" >
      <item>
       <layout class="QHBoxLayout" >
        <item>
         <widget class="QLabel" name="hscaleLabel_" >
          <property name="maximumSize" >
           <size>
            <width>22</width>
            <height>22</height>
           </size>
          </property>
          <property name="text" >
           <string>TextLabel</string>
          </property>
          <property name="buddy" >
           <cstring>fontHScale_</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="SMScrSpinBox" name="fontHScale_" >
          <property name="suffix" >
           <string> %</string>
          </property>
          <property name="minimum" >
           <number>10</number>
          </property>
          <property name="maximum" >
           <number>400</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="vscaleLabel_" >
          <property name="maximumSize" >
           <size>
            <width>22</width>
            <height>22</height>
           </size>
          </property>
          <property name="text" >
           <string>TextLabel</string>
          </property>
          <property name="buddy" >
           <cstring>fontVScale_</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="SMScrSpinBox" name="fontVScale_" >
          <property name="suffix" >
           <string> %</string>
          </property>
          <property name="minimum" >
           <number>10</number>
          </property>
          <property name="maximum" >
           <number>400</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="baselineOffsetLabel_" >
          <property name="maximumSize" >
           <size>
            <width>22</width>
            <height>22</height>
           </size>
          </property>
          <property name="text" >
           <string>TextLabel</string>
          </property>
          <property name="buddy" >
           <cstring>baselineOffset_</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="SMScrSpinBox" name="baselineOffset_" >
          <property name="suffix" >
           <string> %</string>
          </property>
          <property name="minimum" >
           <number>-100</number>
          </property>
          <property name="maximum" >
           <number>100</number>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" >
        <item>
         <widget class="QLabel" name="languageLabel_" >
          <property name="text" >
           <string>Language:</string>
          </property>
          <property name="buddy" >
           <cstring>language_</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="SMScComboBox" name="language_" />
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <spacer>
        <property name="orientation" >
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType" >
         <enum>QSizePolicy::MinimumExpanding</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>300</width>
          <height>0</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="smColorGroup" >
     <property name="title" >
      <string>Colors</string>
     </property>
     <layout class="QVBoxLayout" >
      <item>
       <layout class="QHBoxLayout" >
        <item>
         <widget class="QLabel" name="FillIcon" >
          <property name="maximumSize" >
           <size>
            <width>22</width>
            <height>22</height>
           </size>
          </property>
          <property name="text" >
           <string>TextLabel</string>
          </property>
          <property name="buddy" >
           <cstring>fillColor_</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="SMColorCombo" name="fillColor_" />
        </item>
        <item>
         <widget class="QLabel" name="fillShadeLabel" >
          <property name="maximumSize" >
           <size>
            <width>22</width>
            <height>22</height>
           </size>
          </property>
          <property name="text" >
           <string>TextLabel</string>
          </property>
          <property name="buddy" >
           <cstring>fillShade_</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="SMShadeButton" name="fillShade_" >
          <property name="text" >
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" >
        <item>
         <widget class="QLabel" name="StrokeIcon" >
          <property name="maximumSize" >
           <size>
            <width>22</width>
            <height>22</height>
           </size>
          </property>
          <property name="text" >
           <string>TextLabel</string>
          </property>
          <property name="buddy" >
           <cstring>strokeColor_</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="SMColorCombo" name="strokeColor_" />
        </item>
        <item>
         <widget class="QLabel" name="strokeShadeLabel" >
          <property name="maximumSize" >
           <size>
            <width>22</width>
            <height>22</height>
           </size>
          </property>
          <property name="text" >
           <string>TextLabel</string>
          </property>
          <property name="buddy" >
           <cstring>strokeShade_</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="SMShadeButton" name="strokeShade_" >
          <property name="text" >
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <spacer>
        <property name="orientation" >
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType" >
         <enum>QSizePolicy::MinimumExpanding</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>72</width>
          <height>0</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <customwidgets>
  <customwidget>
   <class>SMScComboBox</class>
   <extends>QComboBox</extends>
   <header>smsccombobox.h</header>
  </customwidget>
  <customwidget>
   <class>SMScrSpinBox</class>
   <extends>QSpinBox</extends>
   <header>smscrspinbox.h</header>
  </customwidget>
  <customwidget>
   <class>SMColorCombo</class>
   <extends>QComboBox</extends>
   <header>smcolorcombo.h</header>
  </customwidget>
  <customwidget>
   <class>SMShadeButton</class>
   <extends>QPushButton</extends>
   <header>smshadebutton.h</header>
  </customwidget>
  <customwidget>
   <class>SMFontComboH</class>
   <extends>QWidget</extends>
   <header>smfontcomboh.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SMStyleSelect</class>
   <extends>QWidget</extends>
   <header>smstyleselect.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
